src/Umbraco.Core/Composing/ComposerGraph.cs

```diff
@@ -9,36 +9,33 @@
 
 namespace Umbraco.Cms.Core.Composing
 {
-    // note: this class is NOT thread-safe in any ways
+    // note: this class is NOT thread-safe in any way
 
     /// <summary>
     /// Handles the composers.
     /// </summary>
-    public class Composers
+    public class ComposerGraph
```

Comment:
Wonder if we should do this internal. I see no reason users of Umbraco should use this.

Reply:
And renaming a public class is a breaking change anyway... I could see this get used by tests to check the enabled status and order of your own composers, but because this can be influenced by any external assembly (and its attributes), those tests will be of minimal use anyway.